addons/sourcemod/scripting/shavit-wr.sp

```diff
@@ -2209,6 +2282,36 @@ public void Shavit_OnFinish(int client, int style, float time, int jumps, int st
 		#if defined DEBUG
 		Shavit_PrintToChat(client, "old: %.01f new: %.01f", fOldWR, time);
 		#endif
+
+		int length = gA_StageTimes[client][style].Length;
+
+		if(length != 0)
+		{
+			char stageTimes[128];
+
+			for(int i = 0; i < length; i++)
+			{
+				char sStageTime[32];
+				FormatSeconds(gA_StageTimes[client][style].Get(i), sStageTime, 32);
+
+				if(i == 0)
+				{
+					FormatEx(stageTimes, sizeof(stageTimes), "%s", sStageTime);
+				}
+				else
+				{
+					FormatEx(stageTimes, sizeof(stageTimes), "%s;%s", stageTimes, sStageTime);
```

Comment:
"%s;%s" to "%s:%s"

Reply:
no, the times are already formatted with FormatSeconds so using a colon here would break separating stage times

Reply:
no, this is a string to yet get formatted, you could put "%sLOL%s" there and the result would be stageTimesLOLsStageTime. This is stageTimes;sStageTime

Reply:
and they're from FormatSeconds() like 8 lines up

Reply:
also that line should use Format() instead of FormatEx() because overlapping input/output buffers